pom.xml

```diff
@@ -86,7 +86,8 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
-      <version>1.89</version>
+      <classifier>shaded</classifier>
```

Comment:
some conflict when running the latest version, getting a repository that didn't exist seemed to go into an infinite loop.

couldn't find the conflict, but this fixed it